pages/api/cardinfo/class.ts

```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import { getClassInfoWithProgramId } from "@database/programcardinfo";
+
+/**
+ * handle takes the programId parameter and returns
+ * the program associated with the programId
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "GET") {
+        const { id } = req.query;
+        const result = await getClassInfoWithProgramId(id as string);
```

Comment:
I think we should just create an endpoint in `api/class/index.ts` for this. We should return classes by programId in the backend and reutrn everything about each class to the frontend. We should let the frontend just filter/process the data to the way that's required.